lua/entities/dak_turretmotor/init.lua

```diff
@@ -1,199 +1,136 @@
 AddCSLuaFile("cl_init.lua")
 AddCSLuaFile("shared.lua")
-
 include("shared.lua")
-
 local DTTE = DTTE
-
 ENT.DakEngine = NULL
 ENT.DakMaxHealth = 10
 ENT.DakHealth = 10
---ENT.DakName = "TMotor"
---ENT.DakModel = "models/xqm/hydcontrolbox.mdl"
 ENT.DakMass = 250
 ENT.DakPooled = 0
-
 function ENT:Initialize()
-	--self:SetModel(self.DakModel)
 	self:PhysicsInit(SOLID_VPHYSICS)
 	self:SetMoveType(MOVETYPE_VPHYSICS)
 	self:SetSolid(SOLID_VPHYSICS)
 	self.DakArmor = 10
 	self.DakHealth = self.DakMaxHealth
-	--local phys = self:GetPhysicsObject()
-
-	self.Soundtime = CurTime()
 	self.SparkTime = CurTime()
 	self.DakBurnStacks = 0
 end
 
 function ENT:Think()
+	local self = self
+	local selfTbl = self:GetTable()
+
 	DTTE.CheckSpherical(self)
-	if CurTime() >= self.SparkTime + 0.33 then
-		if self.DakHealth<=(self.DakMaxHealth*0.80) and self.DakHealth>(self.DakMaxHealth*0.60) then
-			local effectdata = EffectData()
-			effectdata:SetOrigin(self:GetPos())
-			effectdata:SetEntity(self)
-			effectdata:SetAttachment(1)
-			effectdata:SetMagnitude(.5)
-			effectdata:SetScale(1)
-			util.Effect("daktedamage", effectdata)
-			if CurTime()>=self.Soundtime+3 then
-				self:EmitSound( "dak/shock.wav", 60, math.Rand(60,150), 0.4, 6)
```

Comment:
I did. The part that plays that shock sound was present, but commented out, in every other file. So I assumed Dakota probably left it in accidentally. In any case I don't think it had a notable impact on gameplay anyway. And I'd like parts that are repeated to actually be the same if possible, so that it's easier to make a base-class for daktank components in the far future and cut down on repeated/redundant code a bit more.

Reply:
Understandable; I'll go ahead and merge then since this was the only issue that I noticed.